generated/gitops-template/githubactions/.github/workflows/gitops-promotion.yml

```diff
@@ -60,33 +62,47 @@ jobs:
       uses: actions/github-script@v7
       with:
         script: |
-          const secrets = {
-            IMAGE_REGISTRY: `${{ secrets.IMAGE_REGISTRY }}`,
+          const vars = {
+            IMAGE_REGISTRY: `${{ vars.IMAGE_REGISTRY }}`,
```

Comment:
Outside the scope of this PR, but I think this check would be more realistic if it actually just read the values from the environment variables that are configured at the top of this file. Do you know the reason why that's not the case?

Even further, it would be great to just have another bash script in the runner image to do this verification instead of embedding a script here.